src/app/containers/Footer/index.test.jsx

```diff
@@ -59,15 +66,49 @@ describe(`FooterContainer`, () => {
 
   shouldMatchSnapshot(
     'should render correctly',
-    <ServiceContext.Provider value={contextStub}>
-      <FooterContainer />
-    </ServiceContext.Provider>,
+    FooterWithContext(contextStub),
   );
 
   isNull(
-    'should render null when footer config not availible',
-    <ServiceContext.Provider value={{}}>
-      <FooterContainer />
-    </ServiceContext.Provider>,
+    'should render null when footer config not available',
+    FooterWithContext({}),
   );
+
+  describe('assertions', () => {
+    it('should contain copyright text', () => {
+      const { container } = render(FooterWithContext(contextStub));
+
+      const paragraph = container.getElementsByTagName('p')[0];
```

Comment:
Could we add a `data-e2e` attribute to the copyright text element? that way we wouldn't have to look for a p tag at index 0

Reply:
That would be the perfect scenario, so we won't have to be removing the other elements from the paragraph to get the right text, but the footer is using the Psammead `Sitewidelinks` component so that would require to add it Psammead first to be able to use it here. Also, isn't `data-e2e` specific for e2e test, not unit/integration tests?

Reply:
The label can be anything (Ive just had e2e in my head for ages now :P) IIRC react-testing-library automatically looks for an attribute of `data-testid` happy for this to go as is, however I think we need to ensure we have one standard way of getting dom elements. 

I thought we could pass any prop to a psammed component and it would spread them all?

Reply:
Unfortunately, that component was built time ago and it doesn't allow to pass more props down. Also, the paragraph is inside of it, so I believe we can' target it directly with an attribute.

```
const SitewideLinks = ({
  links,
  trustProjectLink,
  copyrightText,
  externalLink,
  service,
}) => (
  <SitewideLinksWrapper service={service}>
    <ConstrainedWrapper trustProjectLink={trustProjectLink}>
      <List links={links} trustProjectLink={trustProjectLink} />
      <StyledParagraph>
        {copyrightText}{' '}
        <Link text={externalLink.text} href={externalLink.href} inline />
      </StyledParagraph>
    </ConstrainedWrapper>
  </SitewideLinksWrapper>
);
```

Reply:
Ok thanks for the info @DenisHdz